client/src/settings/tasks/types.js

```diff
@@ -5,7 +5,8 @@ import {TaskType, PythonSubtypes} from "../../../../shared/tasks";
 const taskSubtypeLabels = {
     [TaskType.PYTHON]: {
         [PythonSubtypes.ENERGY_PLUS]: 'EnergyPlus task',
-        [PythonSubtypes.NUMPY]: 'Numpy task'
+        [PythonSubtypes.NUMPY]: 'Numpy task',
+        [PythonSubtypes.ANOMALY]: 'Anomaly task'
```

Comment:
Somehow forgot to add this in https://github.com/bastianluk/ivis-core/pull/14. 🤦